src/components/Download/DownloadOptionsForm/index.js

```diff
@@ -21,22 +22,24 @@ export const DownloadOptionsForm = ({
   const { updateDataset, getDownloadOptions, updateDownloadOptions } =
     useDatasetManager()
   const { setResponsive } = useResponsive()
-  const isProcessed = dataset?.is_processed
+  const [isRegenerate, setIsRegenerate] = useState(false)
```

Comment:
you can use `canRegenerate` here if you want to capture potential state instead of current state.

Reply:
This is better 👍!